docs/reference/aggregations/bucket/terms-aggregation.asciidoc

```diff
@@ -617,7 +621,7 @@ mode as opposed to the default `depth_first` mode:
              "terms" : {
                  "field" : "actors",
                  "size" : 10,
-                 "collect_mode" : "breadth_first"
+                 "collect_mode" : "breadth_first|depth_first"
```

Comment:
I think we should keep it a working example, so intead of putting `breadth_first|depth_first`, maybe we can add an annotation that says that it also accepts `depth_first`?